docs/src/less/highlight.less

```diff
@@ -19,3 +19,15 @@ figure.highlight {
         white-space: pre-wrap;
     }
 }
+
+@media (max-width: 600px) {
+    a,
```

Comment:
I don't think we should be adding global rules inside highlight.less. Seems like the wrong place for these. Same goes for the the `demo` selector. I'd actually prefer not have any of this as part of the PR if possible, as it seems unrelated. I'm guessing they help for the dialog examples, but I'm not sure what effect it's going to have in other places.

Reply:
So ... this was for overflows that were breaking the page on smaller screens, and it's specific to the highlighting of the code.

And, I could not test the dialog properly without these changes in place.

Reply:
Ok, but they shouldn't live in highlight.less. That file is specific to the Pygments/Jekyll stuff.

Reply:
These rules are far reaching, so if there's any way to make it more specific to dialog, that would be good. Not sure we really want all h2s and anchors to have break word.

Reply:
Removed.